src/pages/home/Home.jsx

```diff
@@ -1,12 +1,59 @@
 import axios from "axios";
 import React from "react";
 import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useCategories } from "../../context/categories-context";
+import {ImageSlider} from "../../components/ImageSlider/ImageSlider";
```

Comment:
put the image in public folder then access it as a URL , read on accessing files in public folder in react